### PR TITLE
Support setting multiple breakpoints and set breakpoints on load

### PR DESCRIPTION
**Pull request by cquiroz**
breakpoints are read from the odb on load and it is possible to set multiple breakpoints at once in a separate `breakpoints` route

### CODE REVIEW

File: modules/engine/src/main/scala/observe/engine/Event.scala [hunk 1]
```diff
@@ -33,11 +34,11 @@ object Event {
   def cancelPause[F[_], S, U](id: Observation.Id, user: User): Event[F, S, U]               =
     EventUser[F, S, U](CancelPause(id, user.some))
   def breakpoint[F[_], S, U](
```
Comment (rpiaggio) [inline modules/engine/src/main/scala/observe/engine/Event.scala:36]: `def breakpoints` ? (with s)

Comment (cquiroz) [reply modules/engine/src/main/scala/observe/engine/Event.scala:36]: yes, it takes a list

File: modules/model/shared/src/main/scala/observe/model/Step.scala [hunk 2]
```diff
@@ -32,6 +33,10 @@ object Step {
   extension [A](l: Lens[A, Boolean]) {
     def negate: A => A = l.modify(!_)
```
Comment (rpiaggio) [inline modules/model/shared/src/main/scala/observe/model/Step.scala:34]: Do we still need `negate` for other things?

File: modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala [hunk 3]
```diff
@@ -71,11 +73,20 @@ class ObserveCommandRoutes[F[_]: Async: Compression](
...
       ssoClient.require(req) { user =>
-        oe.setBreakpoint(obsId, user, obs, stepId, bp) *>
+        oe.setBreakpoints(obsId, user, obs, List(stepId), bp) *>
           NoContent()
       }
+
+    case req @ POST -> Root / ObsIdVar(obsId) / ClientIDVar(clientId) /
+        "breakpoints" / ObserverVar(obs) / BreakpointVar(bp) =>
+      ssoClient.require(req) { user =>
+        req.decode[List[Step.Id]] { steps =>
```
Comment (rpiaggio) [inline modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala:85]: How is this decoded? Just a comma-separated list of step ids?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: modules/model/shared/src/main/scala/observe/model/Step.scala (hunk 2 above)
Comment (cquiroz) [reply modules/model/shared/src/main/scala/observe/model/Step.scala:34]: it is being called bet we'll probablyy remove it when we take the skipmark off

File: modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala (hunk 3 above)
Comment (cquiroz) [reply modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala:85]: Yes like `["s-f5efec44-32f4-3b6b-8e9a-39dcf55ed31c", "s-800d515f-835d-3522-aca3-1f21b2865d84"]`

**Pull request merged**